numpy/core/src/private/npy_binsearch.h.src

```diff
@@ -100,10 +102,9 @@ static PyArray_@Arg@BinSearchFunc *gen@arg@binsearch_map[] = {
 static NPY_INLINE PyArray_@Arg@BinSearchFunc*
 get_@arg@binsearch_func(PyArray_Descr *dtype, NPY_SEARCHSIDE side)
 {
-    static npy_intp num_funcs = sizeof(_@arg@binsearch_map) /
```

Comment:
Interesting that this was previously static

Reply:
That was my thought. Even if made constant, accessing data memory tends to be slower than accessing the stack. I expect the compiler to optimize the whole thing away and use a literal.